doc/api/stream.md

```diff
@@ -860,6 +860,7 @@ added: v0.9.4
 * `destination` {stream.Writable} The destination for writing data
 * `options` {Object} Pipe options
   * `end` {boolean} End the writer when the reader ends. Defaults to `true`.
+* Returns: {stream.Writable} making it possible to set up chains of piped streams
```

Comment:
Sorry, linter issue:
```console
  863:82  warning  Line must be at most 80 characters  maximum-line-length  remark-lint
```

Reply:
my bad
fixed in https://github.com/nodejs/node/pull/19361/commits/125df8375e9d7bd38cfcecfb953d71c85c14dd8c